coalib/results/TextPosition.py

```diff
@@ -31,3 +46,37 @@ def line(self):
     @property
     def column(self):
         return self._column
+
+    @classmethod
+    def from_position(cls, text, pos_to_find, create_object=True):
+        """
+        Makes a TextPosition object or a tuple containing (line, column)
+        by calculating line number and column in the text, from position.
+        Uses \\n as the newline character. Lines and columns start from 1.
+
+        :param text:        A A tuple/list of lines in which position is to
+                            be calculated.
+        :param pos_to_find: Position of character to be found in the
+                            (line, column) form.
+        :create_object:     Boolean which determines whether to return
+                            a TextPosition Object or a tuple.
+        :return:            a TextPosition object or a tuple of the form
```

Comment:
Capital for the starting letter (Also, this `object` does  not start with capital, other does)